pkg/plugin/loader/loader.go

```diff
@@ -75,15 +75,15 @@ func (f *finder) Find() (string, error) {
 		workspacePath := path.Join(cwd, workspaceFilename)
 		if _, err := f.osStat(workspacePath); err != nil {
 			if !os.IsNotExist(err) {
-				return "", fmt.Errorf("failed to find .aspectplugins: %w", err)
+				return "", fmt.Errorf("failed to find Bazel workspace: %w", err)
 			}
 			cwd = filepath.Dir(cwd)
 			continue
 		}
 		aspectpluginsPath := path.Join(cwd, aspectpluginsFilename)
 		if _, err := f.osStat(aspectpluginsPath); err != nil {
 			if !os.IsNotExist(err) {
-				return "", fmt.Errorf("failed to find .aspectplugins: %w", err)
+				return "", fmt.Errorf("failed to find .aspect/cli/plugins.yaml: %w", err)
```

Comment:
nit: you could use the `aspectpluginsFilename` constant here instead of hard-coding the filename in the error msg

Reply:
but we're planning to do another pass of this and consolidate cli/plugins.yaml and cli/config.yaml anyway so no point in perfecting it